### ISSUE TITLE
No AES base alias function .

### ISSUE THREAD

**Issue by haoyu01** [intel/isa-l_crypto#29]
AES function does not implement base functions . As a result , only x86 platform can support AES functions now .
That's different with other functions

I think we should implement base functions

**Comment by gbtucker** [comment]
Correct, it would be nice but note it's a very difficult task to write equal-time base functions in C.

**Comment by haoyu01** [comment]
> 
> 
> Correct, it would be nice but note it's a very difficult task to write equal-time base functions in C.

Is there any method to make sure "equal-time" ?  I am preparing AES job , I must setup some local test firstly

**Comment by IamFive** [comment]
Hi, @haoyu01. you still working on this ? We are trying to add aes implemention using AES instruction set directly.

**Comment by haoyu01** [comment]
@IamFive , I haven't start AES job .  
And I think that's good news for me .

**Comment by IamFive** [comment]
@haoyu01  I am working on this for Huawei, but they did not accept this solution (using AES instruction like `vextq_u8` directly). SO, i give up this task too.
